python/paddle/fluid/dygraph/dygraph_to_static/convert_operators.py

```diff
@@ -0,0 +1,136 @@
+# Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from paddle.fluid.framework import Variable
+from paddle.fluid.layers import control_flow, logical_and, logical_or, logical_not
+from paddle.fluid.dygraph.dygraph_to_static.variable_trans_func import to_static_variable
+
+
+def convert_while_loop(cond, body, loop_vars):
+    """
+    A function representation of a Python ``while`` statement.
+
+    Args:
+        cond(Callable): A callable returning a boolean variable controlling whether to continue looping. And ``cond`` takes
+        as many arguments as ``loop_vars`` .
+        body(Callable): A callable returning a tuple or list of variables of the same arity
+            (length and structure) and types as ``loops_vars`` . And ``body`` takes as many arguments as ``loop_vars`` .
+        loop_vars(list|tuple): A list or tuple of variables that is passed to both ``cond`` and ``body`` .
+
```

Comment:
+ cond(Callable): A callable object that returns a boolean variable to control whether to  execute the loop body.  It takes  ``loop_vars`` as arguments.

+ body(Callable): A callable object that returns a tuple or list of variables with the same arguments ``loops_vars`` as ``cond`` . 

+ loop_vars(list|tuple): A list or tuple of variables passed to ``cond`` and ``body`` .

Reply:
> * cond(Callable): A callable object that returns a boolean variable to control whether to  execute the loop body.  It takes  `loop_vars` as arguments.
> * body(Callable): A callable object that returns a tuple or list of variables with the same arguments `loops_vars` as `cond` .
> * loop_vars(list|tuple): A list or tuple of variables passed to `cond` and `body` .

Thanks, done